src/main/java/lebron/task/Task.java

```diff
@@ -0,0 +1,62 @@
+package lebron.task;
+
+/**
+ * This class represents a Task the user gives to the bot.
+ *
+ * @author Nigel Tan
+ */
+public class Task {
+
+    private String name;
+    private boolean isDone;
+
+    /**
+     * Constructor.
+     *
+     * @param name the name of the task
+     */
+    public Task(String name) {
+        this.name = name;
+        this.isDone = false;
+    }
+
+    /**
+     * This method marks the task as true.
+     */
+    public void mark() {
```

Comment:
Isn't the method name 'mark' slightly more ambiguous than 'markAsDone' for example? It is ambiguous in this case whether mark denotes mark as true or mark as false (unless I read the javadoc or look at the method implementation).